pkg/types/field_export_reconciler.go

```diff
@@ -33,7 +33,7 @@ type FieldExportReconciler interface {
 		context.Context,
 		AWSResource,
 		ackv1alpha1.FieldExport,
-	) error
+	) (ackv1alpha1.FieldExport, error)
```

Comment:
curious: was this return type updated just for the unit testing?

Reply:
Yeah, to be in line with other reconcilers, for unit testing and since we now maintain a "desired" and a "latest" - we are not directly editing the passed argument.